GDJS/GDJS/Events/CodeGeneration/EventsCodeGenerator.cpp

```diff
@@ -1298,7 +1298,20 @@ gd::String EventsCodeGenerator::GenerateGetVariable(
     const gd::String& objectName) {
   gd::String output;
   const gd::VariablesContainer* variables = NULL;
-  if (scope == LAYOUT_VARIABLE) {
+  if (scope == ANY_VARIABLE) {
```

Comment:
It will be interesting to see if we can pass here a VariablesContainersList at some point so the work of scoping is always done by a method of VariablesContainersList.